files/en-us/web/css/forced-color-adjust/index.html

```diff
@@ -0,0 +1,125 @@
+---
+title: forced-color-adjust
+slug: Web/CSS/forced-color-adjust
+tags:
+  - Adjusting Colors
+  - CSS
+  - CSS Colors
+  - CSS Property
+  - HTML Colors
+  - HTML Styles
+  - Layout
+  - Reference
+  - Styling HTML
+  - Styling text
+  - Web
+  - forced-color-adjust
+  - 'recipe:css-property'
+---
+<div>{{CSSRef}}</div>
+
+<p><span class="seoSummary">The <strong><code>forced-color-adjust</code></strong> CSS property allows authors to opt certain elements out of forced colors mode. This then restores the control of those values to CSS.</p>
+
+<h2 id="Syntax">Syntax</h2>
+
+<pre class="syntaxbox notranslate">forced-color-adjust: auto;
+forced-color-adjust: none;</pre>
+
+<p>The <code>forced-color-adjust</code> property's value must be one of the following keywords.</p>
+
+<h3 id="Values">Values</h3>
+
+<dl>
+	<dt><code>auto</code></dt>
+	<dd>The element's colors are adjusted by the {{Glossary("user agent")}} in forced colors mode. This is the default.</dd>
+	<dt><code><strong>none</strong></code></dt>
+	<dd>The element's colors are not automatically adjusted by the {{Glossary("user agent")}} in forced colors mode.</dd>
+</dl>
+
+<h2 id="Usage_notes">Usage notes</h2>
+
+<p>This property should only be used to makes changes that will support a user's color and contrast requirements. For example, if you become aware that the color optimizations made by the {{Glossary("user agent")}} result in a poor experience when in a high contrast or dark mode. Using this property would enable tweaking of the result in that mode to provide a better experience. It should not be used to prevent user choices being respected.</p>
```

Comment:
```suggestion
<p>This property should only be used to makes changes that will support a user's color and contrast requirements. For example, if you become aware that the color optimizations made by the {{Glossary("user agent")}} result in a poor experience when in a high contrast or dark mode. Using this property would enable tweaking of the result in that mode to provide a better experience. <strong>It should not be used to prevent user choices being respected</strong>.</p>
```

Making the main point of this more obvious, since I'm assuming people will skim 😅